go.mod

```diff
@@ -366,3 +366,7 @@ replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday
 // happen, for example, during a read when the sqlite db is under heavy write load.
 // This patch cherry picks compatible fixes from upstream xorm PR#1998 and can be reverted on upgrade to xorm v1.2.0+.
 replace xorm.io/xorm => github.com/grafana/xorm v0.8.3-0.20220614223926-2fcda7565af6
+
+// Use our fork of the upstream alertmanagers.
+// This is required in order to get notification delivery errors from the receivers API.
+replace github.com/prometheus/alertmanager => github.com/grafana/prometheus-alertmanager v0.24.1-0.20221006171631-7f1ce00421dc
```

Comment:
This is different from the original PR where we had `v0.24.0-rc.0.0.20220930143838-d75bdc5543c0`?

Reply:
Yes, there's a test that requires a change in the AM that wasn't present in that commit I referenced in the original PR.
In short, the test required the AM to truncate text using ellipses (`…`) instead of three dots (`...`). This change was not present in our fork until I merged commits from upstream.